android_automation/page_action/home_page.py

```diff
@@ -0,0 +1,236 @@
+from time import sleep
+from appium.webdriver.common.appiumby import AppiumBy
+from selenium.common import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from com_utils.api_control import home_banner_info
+from com_utils.element_control import aal, aalc, aals, scroll_control, swipe_control
+from ios_automation.page_action import context_change
+from ios_automation.page_action.bottom_sheet import close_bottom_sheet
+
+
+def check_home_logo(wd):
+    try:
+        aal(wd, 'navi_logo_btn')
+        print('HOME 탭으로 이동 확인')
+    except NoSuchElementException:
+        print('HOME 탭으로 이동 확인 실패')
+
+
+def click_search_btn(wd):
+    aalc(wd, 'com.the29cm.app29cm:id/imgSearch')
+    print("상단 검색 아이콘 선택")
+    sleep(2)
+
+
+def click_close_life_tab(wd):
+    # 디폴트 선택 화면 확인
+    top_tabs = aal(wd, 'com.the29cm.app29cm:id/tabs')
+    culture_tab = aal(top_tabs, '//android.view.View[@index=4]/android.widget.TextView')
+    if culture_tab.text == '푸드':
+        print("푸드 탭 존재")
+        aalc(wd, 'life_tab')
+        print("라이프 탭 해제")
+    else:
+        print('라이프 탭 디폴트 아님')
+
+    sleep(3)
+
+
+# click_tab_name : 선택하려는 상단 탭 이름 입력
+def click_tab_name(wd, click_tab_name):
+    aalc(wd, click_tab_name)
+    print(f'{click_tab_name} 탭 선택')
+    sleep(2)
+
+
+def save_tab_names(wd):
+    tab = aals(wd, f'//*[contains(@label, "라이프")]/../../../XCUIElementTypeCell')
+    tab_name_list = []
+    for text in tab:
+        tab_name = text.find_element(AppiumBy.XPATH, '//XCUIElementTypeStaticText').text
+        tab_name_list.append(tab_name)
+    tab_name_list = ', '.join(tab_name_list)
+    return tab_name_list
+
+
+# tab : 탭을 보여주는 카테고리 / 전체 : 'home' / 라이프 선택 상태 : 'life'
+# tab_list : 비교할 탭 이름 리스트
+def check_tab_names(self, tab, tab_list):
+    if self.conf['compare_home_tab'][tab] in tab_list:
+        print(f'홈 상단 {tab} 탭 확인')
+    else:
+        print(f'홈 상단 {tab} 탭 확인 실패 - {tab_list}')
+        raise Exception(f'홈 상단 {tab} 탭 확인 실패')
+
+
+def click_dynamic_gate(wd):
+    # 다이나믹 게이트 -> 센스있는 선물하기 선택
+    for i in range(0, 3):
+        try:
+            element = aal(wd, '센스있는 선물하기')
+            if element.is_displayed():
+                aalc(wd, '센스있는 선물하기')
+                sleep(3)
+                break
+            else:
+                dynamic_gate = aal(wd, 'dynamic_gate')
+                swipe_control(wd, dynamic_gate, 'left', 30)
+        except NoSuchElementException:
+            dynamic_gate = aal(wd, 'dynamic_gate')
+            swipe_control(wd, dynamic_gate, 'left', 30)
+
+
+def check_dynamic_gate_gift_page(wd):
+    try:
+        aal(wd, '센스있는 선물하기')
+        context_change.switch_context(wd, 'webview')
```

Comment:
해진님, 해당 코드는 iOS에서 사용하는 방식인데 Android에서도 동작하나요??
Android 기존 코드 참고하면 아래와 같은 코드가 아닐까 생각됩니다.
```suggestion
        context_change.change_webview_context(wd)
```

Reply:
넵 해당 코드도 ios에 있던거라 다음 pr에 제거하도록하겠습니다